packages/frontend/src/Layout/Nav/Nav.styled.ts

```diff
@@ -113,24 +133,50 @@ export const NavItem = styled.li`
   text-align: center;
 `;
 
+export const DesktopNavItems = styled.ul`
+  display: none;
+  @media ${Breakpoint.Tablet} {
+    display: flex;
+    flex-direction: row;
+    margin-bottom: 0;
+    margin-left: 60px;
+    list-style: none;
+  }
+`;
+
 export const StyledLink = styled(Link)<LinkProps>`
-  display: block;
+  display: ${({ $isDesktop }) => ($isDesktop ? "none" : "block")};
   font-size: ${FontSize.Medium};
   color: ${({ $isActive }) => ($isActive ? `${Color.White}` : `${rgba(`${Color.White}`, 0.4)}`)};
   padding: 8px 0;
-  transition: 0.2s;
+  transition: color 0.2s;
   :hover {
     color: ${Color.White};
-    font-size: ${FontSize.Medium};
+  }
+  :nth-child(2) {
```

Comment:
napewno to będzie zawsze dla drugiego elementu jeśli fokusujesz tu ostatni element to użyłbym :last-child jeśli nie to napisz w jakim celu używasz tego selektora